entc/gen/func.go

```diff
@@ -418,7 +418,12 @@ func dumpFields(v interface{}) string {
 		}
 		fv := rv.Field(i)
 		if !fv.IsZero() {
-			fields = append(fields, fmt.Sprintf("%s: %#v", f.Name, fv.Interface()))
+			if fv.Kind() == reflect.Ptr {
+				fv = reflect.Indirect(fv)
```

Comment:
No need to call Indirect, use .Elem() as we already know it's a pointer.
```go
fv = fv.Elem()
```

Reply:
fixed, thx, you learn something new every day :D

Reply:
actually that doesn't work because without doing `fv = reflect.Indirect(fv)` the `fv.Type()` will be `*bool` which results in `&[]*bool{false}[0]}`